app/src/migrations/20200427000000-2-location-worker-count-col.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.addColumn(
+          'location',
+          'numberOfWorkers',
+          {
+            allowNull: false,
+            comment: 'Number of all workers at a location',
+            type: Sequelize.INTEGER,
+            defaultValue: 0,
```

Comment:
After thinking about it, we may need to check with the business that it's ok to have this be non-nullable and default to 0.
Might want to see if they'd want it blank for sites up until the moment the new field is deployed rather than 0?
Don't know if it would cause confusion with inspectors checking sites and seeing a 0 and not knowing if that was entered as 0, or just filled out before the field was deployed...